tests/integration/test.replication.js

```diff
@@ -1568,23 +1568,32 @@ adapters.forEach(function (adapters) {
         result.ok.should.equal(true);
         result.docs_written.should.equal(0);
         result.docs_read.should.equal(0);
-        changesSince.length.should.equal(2);
-        // the returned last_seq should match the 'since'
-        // requested from remote
-        result.last_seq.should.equal(changesSince[1]);
+        changesSince.length.should.equal(2, JSON.stringify(changesSince));
+
+        if (!testUtils.isCouchMaster()) {
```

Comment:
I think we should take out these assertions, if they dont hold for couch master then we dont strictly need them, I am good with making as many assertions as we can but lets keep them to the shared ones that are true regardless of the server